tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/Agentic.cs

```diff
@@ -16,20 +16,22 @@ namespace Microsoft.Identity.Test.Integration.HeadlessTests
     [TestClass]
     public class Agentic
     {
-        const string ClientId = "d15884b6-a447-4dd5-a5a5-a668c49f6300"; // agent app
+        const string ClientId = "d05619c9-dbf2-4e60-95fd-cc75dd0db451"; // agent app
```

Comment:
Did you try locally, @bgavrilMS ?
This tenant was de-authed the night after my idweb PR